native/packages/react-native-bpk-component-icon/src/__snapshots__/BpkIcon-test.android.js.snap

```diff
@@ -66,6 +66,6 @@ exports[`Android BpkIcon should render small icon correctly 1`] = `
     ]
   }
 >
-  
+  
```

Comment:
Is it me or has nothing changed here?

Reply:
I think the unicode actually changed but the browser can't render it

Reply:
Ahh ok